frontend/public/components/_monitoring.scss

```diff
@@ -8,6 +8,13 @@
   margin-left: 14px;
 }
 
+.monitoring-grid-head {
+  color: $color-text-secondary;
```

Comment:
In order to keep things consistent with the key/value editor [1], I recommend using some helper classes in the markup instead of all the declarations here.

So this can become:

```
.monitoring-grid-head {
  padding-bottom: 5px;
}
```

plus the proposed markup changes below.

[1] 
![screen shot 2018-11-16 at 11 34 34 am](https://user-images.githubusercontent.com/895728/48634668-16197980-e994-11e8-930d-826f05b5869f.png)

Reply:
Thanks @rhamilto. I changed the definition of `.monitoring-grid-head` as you suggested.